python/paddle/fluid/tests/unittests/ngraph/test_pool2d_ngraph_op.py

```diff
@@ -13,62 +13,247 @@
 # limitations under the License.
 
 from __future__ import print_function
+from __future__ import division
 
-from paddle.fluid.tests.unittests.test_pool2d_op import TestPool2D_Op, TestCase1, TestCase2, TestCase3, TestCase4, TestCase5
+import unittest
+import numpy as np
 
+import paddle.fluid.core as core
+from paddle.fluid.tests.unittests.op_test import OpTest
 
-class TestNGRAPHPool2D_Op(TestPool2D_Op):
-    def setUp(self):
-        super(TestNGRAPHPool2D_Op, self).setUp()
-        self._cpu_only = True
 
-    def init_test_case(self):
-        super(TestNGRAPHPool2D_Op, self).init_test_case()
+def adaptive_start_index(index, input_size, output_size):
```

Comment:
1. Why not use `super(TestNGRAPHxxx_Op, self).setUp()`?
2. Many codes in `test_pool2d_ngraph_op.py` are duplicated in `test_pool2d_op.py`. For simplify, you can refer to `test_pool2d_int8_mkldnn_op.py`. Same for the other three files.